install/operator/pkg/syndesis/action/upgrade.go

```diff
@@ -92,10 +97,28 @@ func (a *upgrade) Execute(scheme *runtime.Scheme, cl Client, syndesis *v1alpha1.
 			for _, res := range resources {
 				operation.SetNamespaceAndOwnerReference(res, syndesis)
 
-				err = createOrReplaceForce(cl, res, true)
+				err = createOrReplaceForce(ctx, a.client, res, true)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Install addons
+			if addonsDir := *configuration.AddonsDirLocation; len(addonsDir) > 0 {
```

Comment:
Looks like the install addons code got duplicated :)

Reply:
Good catch !